staging/src/k8s.io/api/core/v1/well_known_labels.go

```diff
@@ -19,6 +19,9 @@ package v1
 const (
 	LabelHostname = "kubernetes.io/hostname"
 
+	// Label value is the network location of kube-apiserver stored as <ip:port>
```

Comment:
can we also document on what API objects this is used?

Reply:
done